### DISCUSSION TITLE
pls help me guys, i dont know how to fix that :(

### DISCUSSION THREAD

**Discussion by Carocuongbui** [community/community#102494] [Programming Help] [answered]
Help me guys, i dont know how to fix that

![2024-01-28 (1)](https://github.com/community/community/assets/152510064/48780b95-ceb9-456e-a658-0a9a403e4808)

![2024-01-28](https://github.com/community/community/assets/152510064/9246e8f1-d2bc-4759-b8cc-dc9bf6022892)

### Guidelines

- [ ] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by SaraRasoulian** [comment]
I think you must have only 1 index.html file in root folder

**Reply by Carocuongbui** [reply]
;-; how to do that

**Comment by SaraRasoulian** [comment, marked as answer]
> ;-; how to do that

I created [a fork](https://github.com/SaraRasoulian/buicaocuong46cntt1.github.io) of your repository and moved one of your index.html files out of its current folder and into the main folder of your repository. and deployed it using GitHub pages, now you can see the main page is visible but you need to update links, I also recommend renaming your folders 
I'll remove the fork when you solved your issue

**Reply by Carocuongbui** [reply]
ok, i will try it

**Reply by Carocuongbui** [reply]
omg, that run, i love uuuuuuu, Thank you very much

**Reply by SaraRasoulian** [reply]
you're welcome :)
